dashboard/modules/test/test_head.py

```diff
@@ -93,5 +93,11 @@ async def test_aiohttp_cache_lru(self, req) -> aiohttp.web.Response:
         return dashboard_utils.rest_response(
             success=True, message="OK", value=value, timestamp=time.time())
 
+    @routes.get("/test/file")
+    async def test_file(self, req) -> aiohttp.web.FileResponse:
```

Comment:
This is only used for tests, right? better just patch this in tests, no need to put it here.

Reply:
The test module is only enabled in the test case.